queries.sql

```diff
@@ -46,3 +46,38 @@ SELECT name,AVG(weight_kg) FROM animals GROUP BY name;
 SELECT neutered, COUNT(*) FROM animals GROUP BY neutered ORDER BY COUNT(*) DESC LIMIT 1;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+
+SELECT name, full_name
+FROM animals A
+JOIN owners O ON A.owner_id = O.id
+WHERE O.full_name = 'Melody Pond';
+
+SELECT A.name, S.name
+FROM animals A
+JOIN species S ON A.species_id = S.id
+WHERE S.name = 'Pokemon';
+
+
+SELECT O.full_name, A.name
+FROM owners O
+LEFT JOIN animals A ON O.id = A.owner_id;
+
+SELECT S.name, COUNT(A.name)
+FROM species S
+JOIN animals A ON S.id = A.species_id 
+GROUP BY S.name;
+
+
+SELECT A.name,O.full_name
+FROM animals A
+JOIN species S ON A.species_id = S.id
+JOIN owners O ON A.owner_id = O.id
+WHERE S.name = 'Digimon' AND O.full_name = 'Jennifer Orwell';
+
+
```

Comment:
- [ ] Queries for _Who owns the most animals?_ is missing! Kindly ensure to include them, which should be similar to:

```sql
SELECT owners.full_name, COUNT(owners.full_name) FROM animals
   LEFT JOIN owners
   ON animals.species_id = owners.id
   GROUP BY owners.full_name
   ORDER BY COUNT(owners.full_name) DESC
```